modules/snippy/core/main.nf

```diff
@@ -0,0 +1,53 @@
+process SNIPPY_CORE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::snippy=4.6.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/snippy:4.6.0--hdfd78af_2':
+        'quay.io/biocontainers/snippy:4.6.0--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(vcf), path(aligned_fa)
+    path reference
+
+    output:
+    tuple val(meta), path("${prefix}.aln")     , emit: aln
+    tuple val(meta), path("${prefix}.full.aln"), emit: full_aln
+    tuple val(meta), path("${prefix}.tab")     , emit: tab
+    tuple val(meta), path("${prefix}.vcf")     , emit: vcf
+    tuple val(meta), path("${prefix}.txt")     , emit: txt
+    path "versions.yml"                        , emit: versions
```

Comment:
Ideally these would be gzipped, or does this block downstream snippy tools/steps?

Reply:
Ah, well I would argue (given this is specific to a 'pipeline' of tools, so I'm guessing you're unlikely to use the output of snippy-core outside of `snippy`) you don't need to compress if the next step doesn't accept it.

Reply:
Sound good, thank you